src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
+                    <option>Option 2</option>
+                    <option>Option 3</option>
+                  </select>
+                </div>
+                <div class="chi-form__item -mb--2">
+                  <label class="chi-label" for="example-ba14">Label</label>
+                  <select class="chi-select" id="example-ba14">
+                    <option value="">Select</option>
+                    <option>Option 1</option>
+                    <option>Option 2</option>
+                    <option>Option 3</option>
+                  </select>
+                </div>
+              </div>
+              <div class="-d--flex -justify-content--center -pb--2">
+                <button class="chi-button -primary -uppercase -px--4">Apply</button>
+                <button class="chi-button -primary -outline -bg--white -uppercase -px--4 -ml--2">Cancel</button>
               </div>
-              <div class="chi-toolbar__filters-mobile">
-                <button class="chi-button -icon -flat chi-drawer__trigger" id="drawer-trigger-br1" data-target="#drawer-1" aria-label="Button action">
+            </div>
+          </div>
+        </div>
+        <div class="chi-backdrop -closed">
+          <div class="chi-backdrop__wrapper">
+            <div class="chi-drawer -right -menu -position--absolute" id="drawer-2">
+              <div class="chi-drawer__header">
+                <span class="chi-drawer__title">Actions</span>
+                <button class="chi-button -icon -close" aria-label="Close">
                   <div class="chi-button__content">
-                    <i class="chi-icon icon-filter"></i>
+                    <i class="chi-icon icon-x"></i>
                   </div>
                 </button>
               </div>
+              <div class="chi-drawer__content -px--2">
+                <ul class="-no-style">
+                  <li class="-text--md">Download data</li>
+                  <div class="chi-divider"></div>
+                  <li class="-text--md">Refresh results</li>
+                  <div class="chi-divider"></div>
+                </ul>
+              </div>
             </div>
           </div>
-          <div class="chi-toolbar__end">
-            <div class="chi-toolbar__actions">
-              <div class="chi-toolbar__actions-desktop">
-                <button class="chi-button -icon -flat" id="button-br-download" aria-label="Download" data-tooltip="Download" data-position="top">
-                  <div class="chi-button__content">
-                    <i class="chi-icon icon-arrow-to-bottom"></i>
+        </div>
+
+        <!-- Javascript -->
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-br1'));
+          chi.drawer(document.getElementById('drawer-trigger-br2'));
+          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
+        </script>
+
+.-portal--show
+  h3 Portal base
+  .example.-mb--3
+    .-p--3
+      .chi-toolbar.-portal
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.